io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AbstractAnalyseTable.java

```diff
@@ -136,17 +139,15 @@ protected void analyseHeader(Sheet sheet) {
 	protected abstract void headerCell(int num, String text);
 
 	/**
-	 * Analyse the table content.
+	 * Analyse the table content
 	 * 
 	 * @param sheet the table sheet
 	 */
 	private void analyseContent(Sheet sheet) {
 		// for each row starting from the second
 		for (int i = 1; i <= sheet.getLastRowNum(); i++) {
```

Comment:
Shouldn't the start be based on the `skipNLines` setting?
The current behavior (skipping one row) I would only expect for `skipNLines == 1`.